body.tex

```diff
@@ -163,10 +163,10 @@ \section*{Scientific Python ecosystem}
 IPython or Jupyter, is ideally suited to exploratory data analysis.
 Users fluidly inspect, manipulate, and visualize their data, and
 rapidly iterate to refine programming statements. These statements are
-then stitched together into imperative or functional programs, or
-narrative computational notebooks.
+then stitched together into imperative or functional programs or
```

Comment:
Will retain the comma here, to group imperative or functional programs together.